src/app/map/leaflet-map/leaflet-map.component.ts

```diff
@@ -165,6 +122,19 @@ export class LeafletMapComponent implements OnInit, AfterViewInit, OnDestroy {
     this.map.remove();
   }
 
+
+  // called when apps list changes
+  ngOnChanges(changes: SimpleChanges) {
+    if (changes.projects && !changes.projects.firstChange && changes.projects.currentValue) {
+
+      const deletedApps = changes.projects.previousValue.filter(x => !changes.projects.currentValue.includes(x)) as Array<Project>;
```

Comment:
Do we need a check to see if `previousValue` exists must like `currentValue` is checked?

Reply:
added check